docker/build.sh

```diff
@@ -52,6 +52,12 @@ if [ $BUILD_MODE == "sgx" ]; then
     ftxsgx-elf2sgxs $EDP_TARGET_DIR/tx-query2-enclave-app --heap-size 0x2000000 --stack-size 0x80000 --threads 6 $EDP_ARGS
     sgxs-sign --key chain-tx-enclave/tx-validation/enclave/Enclave_private.pem $EDP_TARGET_DIR/tx-query2-enclave-app.sgxs $EDP_TARGET_DIR/tx-query2-enclave-app.sig -d --xfrm 7/0 --isvprodid 0 --isvsvn 0
 
+    # tdbe
+    cargo build --package tdb-app-runner
+    RUSTFLAGS="-Ctarget-feature=+aes,+sse2,+sse4.1,+ssse3,+pclmul" cargo build --target x86_64-fortanix-unknown-sgx --package tdb-enclave-app
+    ftxsgx-elf2sgxs $EDP_TARGET_DIR/tdb-enclave-app --heap-size 0x2000000 --stack-size 0x80000 --threads 6 $EDP_ARGS
```

Comment:
are such large stack sizes needed?
in default Enclave.config.xml, it's 0x40000

Reply:
This is what we use for `tx-query`. We can try and optimise these sizes later.

Reply:
but is that necessary? it'll crash with 0x40000?

Reply:
I'll test if it works with smaller heap size.